testsuite/features/build_validation/init_clients/proxy_container.feature

```diff
@@ -29,10 +29,10 @@ Feature: Setup containerized proxy
     And I click on "Bootstrap"
     And I wait until I see "Bootstrap process initiated." text
 
-  # workaround for bsc#1218146
-  Scenario: Reboot the proxy host
-    When I reboot the "proxy" minion through SSH
-    And I wait until port "22" is listening on "proxy" host
+#  workaround for bsc#1218146, once we start using SLE/Leap Micro https://github.com/SUSE/spacewalk/issues/23811
+#  Scenario: Reboot the proxy host
+#    When I reboot the "proxy" minion through SSH
+#    And I wait until port "22" is listening on "proxy" host
```

Comment:
I think we need to differentiate here, since in the 5.0 BV where we use HEAD, the containerized proxy will be a SLE Micro Minion.

Reply:
I see. Ok, so it only affects Uyuni Containerized Proxy. 👍🏼 thanks.